docs/source/index.rst

```diff
@@ -43,6 +43,7 @@ Contents:
    :maxdepth: 2
 
    Overview and Installation <README>
+   Retworkx Tutorials and Guides <tutorial/index>
```

Comment:
I cannot comment, but on line 51 also change the capitalization of NetworkX like you did in the other file

Reply:
Fixed in: https://github.com/Qiskit/retworkx/pull/537/commits/b3fe6f090ad0b6e2e79348cbf626fa3feb7efce8